documentation/howto/import_FH_app.adoc

```diff
@@ -0,0 +1,189 @@
+. In the Features Available` list, select the `Hybrid Mobile Tools + CordovaSim` check box and then click `Install/Update`.
+
+.Hybrid Mobile Tools + CordovaSim Check Box Selected
+image::hybrid_mobile_tools__cordovasim_check_box_selected_1.png[Hybrid Mobile Tools + CordovaSim Check Box Selected]
+
+. Follow the on-screen instructions to complete the installation.
+
+[[set_preferences]]
+=== Set the Preferences for your application import
+To set the preferences:
+
+. Click `Window` > `Preferences`.
+. In the `Preferences` window, in the search field type, `FeedHenry` and press Enter.
+. Complete the following fields in the `Preferences` window:
+** Ensure that the `Target URL` field displays the URL to your FeedHenry server. The URL to your FeedHenry server should be something like: `https://_[your-studio-domain]_.feedhenry.com`. 
+** In the `API Key` field, copy and paste the API key from the FeedHenry website.
+
+.Setting Preferences for FeedHenry
+image::preferences.png[Setting Preferences for FeedHenry]
+
+*Note:* The user can either use an existing API key or generate a new one to set the preferences. To obtain the API key from the FeedHenry website click `Settings` > `API Key Management`. To generate a new API key, click `Settings` > `API Key Management`, and then click `Add New Key`. 
+
+[start=4]
+. Click `Apply` and then click `OK` to close the `Preferences` window.
+
+
+[[import_app]]
+== 1. Import Your FeedHenry Application
+
+Ensure that your JBoss Developer Studio preferences are set before importing a FeedHenry application. If not set yet, JBoss Developer Studio will prompt you to set the preferences. These preferences are set once when importing an application for the first time and the configured preferences are used for all imports in the future. 
+
+To import your FeedHenry application:
+
+. Click `File` > `Import`.
+. Expand `FeedHenry`, click Import Cordova Application, and click `Next`.
+. The `Import` wizard displays the projects that you can import into JBoss Developer Studio. Expand the project to import an application from and select the FeedHenry application to be imported.
+. In the `Directory` field, enter the location where you want to clone the application locally.
+. Click `Finish`.
+   
+.Selecting an Application to be Imported
+image::selecting_FeedHenry_app_to_be_imported.png[Selecting an Application to be Imported]
+
+If you have set a password for the SSH keys, the IDE will prompt you to enter the password so that it can import the application.
+
+*Result:* The FeedHenry project is successfully imported into JBoss Developer Studio and appears in the `Project Explorer` view. The `Cordova config.xml` file for this project opens in the Editor.
+
+=== Test the Application Import 
+To test the application import:
+
+. In the `Project Explore` view, right-click the application and then click `Run As` > `Run w/remote FeedHenry server`.  A CordovaSim simulated device displaying the application appears.
```

Comment:
For UI interactions like keyboard short cuts, button clicks and menu paths asciidoc provides macros for this content.

http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#ui-macros

So the above could be written as menu:Context[Run As > Run w/remote FeedHenry server] and it would be formatted consistently across all documents.